main.go

```diff
@@ -65,13 +65,18 @@ func newManager() (ctrl.Manager, error) {
 		enableLeaderElection bool
 		probeAddr            string
 		portAddr             = 9443
+		leaderElectionID     string
 	)
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&leaderElectionID, "leader-election-id", "ae40cfcb.openshift.io",
+		"The leader election ID used by the controller manager if leader election is enabled.")
+	flag.StringVar(&controllerType, "controller-type", "invalid",
+		"The type of controller to execute by the controller manager.")
```

Comment:
I said elsewhere that this flag should be moved to RamenConfig.  However, for ease of `make run`, I believe we can override RamenConfig with a command line flag.

Reply:
`make run` can take a `--config=...` option, so I would look at this as a config only option even in the future.